certification/internal/shell/base_on_ubi_test.go

```diff
@@ -1,41 +1,38 @@
 package shell
 
 import (
+	"strings"
+
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 )
 
 var _ = Describe("BaseOnUBI", func() {
 	var (
-		engine      FakePodmanEngine
 		base_on_ubi BaseOnUBICheck
+		lines       []string
 	)
 
 	BeforeEach(func() {
-		engine = FakePodmanEngine{
-			RunReportStdout: `ID="rhel"
+		lines = strings.Split(`ID="rhel"
 NAME="Red Hat Enterprise Linux"
-`,
-			RunReportStderr: "",
-		}
+`, "\n")
 	})
 
 	Describe("Checking for UBI as a base", func() {
 		Context("When it is based on UBI", func() {
 			It("should succeed the check", func() {
-				ok, err := base_on_ubi.validate(engine, "dummy/image", logger)
+				ok, err := base_on_ubi.checkBasedOnUbi(lines)
```

Comment:
Prefer the `validate` name as mentioned in an earlier comment. Just leaving this comment here so we consciously address it or leave it be (based on whatever choice is made).

Reply:
I think I've addressed all of this.